server/scripts/standalone/setup-standalone-server.sh

```diff
@@ -108,8 +125,8 @@ PACKAGES=(
 
 ZIG_VERSION=0.11.0
 ZIG_PATH_0_11=${PLATFORMSDK_DIR}/zig-0-11
-ZIG_PACKAGE_NAME=zig-macos-x86_64-${ZIG_VERSION}-dev.3937+78eb3c561.tar.xz
-ZIG_URL=https://ziglang.org/builds
+ZIG_PACKAGE_NAME=zig-macos-x86_64-${ZIG_VERSION}.tar.xz
```

Comment:
Old archive was no more (403), but it still happily produced an empty folder.
So we didn't notice it until running the server.